server/src/core/get.rs

```diff
@@ -0,0 +1,23 @@
+use actix_web::{get, web, HttpResponse};
+use rusqlite::{params, Connection};
+
+#[get("/get/{id}")]
+pub async fn get_handler(params: web::Path<String>) -> HttpResponse {
```

Comment:
I plucked this from the main, since it is not a five-liner status thing, but executes actual code.